src/kontrol/solc_to_k.py

```diff
@@ -327,6 +327,7 @@ class StorageField(NamedTuple):
     data_type: str
     slot: int
     offset: int
+    interface_for: str | None
```

Comment:
I would rename this to something like `interface_name`, or `linked_interface`.

Reply:
Thanks, renamed in https://github.com/runtimeverification/kontrol/pull/628/commits/520cf68e668d40a18828146b91ecf7e95171c471 to `linked_interface`!